05-how-to-contribute-to-open-source.md

```diff
@@ -214,35 +200,21 @@ You'll need to have these tools donwloaded and installed on your local machine:
 
    > **Tip**: If you don't see your profile in the "Contributors" section, zoom out the screen until you can see it before taking a screenshot.
 
-9. If you used the CLI tool method in step 5 to add yourself to the guestbook, **skip this step**. Otherwise:
-
-   Add and commit your changes by running the following command:
-
-   ```bash
-   git commit -am "Your commit message"
-   ```
-
-   Replace `"Your commit message"` to your own message. For example, `"Add Alice as a contributor"`.
+9. Run `git log` to check if your changes have been automatically committed. Here's what you should expect to see as your commit message:
 
-   > **Note**: Unlike the `git add .` and `git commit -m "Your message"` approach, in the `git commit -am` command, adding `-a` flag to `git commit` skips the staging process. The `-a` flag automatically stages all your changes in _committed_ files. Combining it with the `-m` flag allows you to write the commit message.
+   `docs: add @your_username as a contributor`
 
-10. Run `git log` to check if you have committed your changes. Press `Q` to close the log.
+   Then, press `Q` to run the log.
```

Comment:
- Let's move the instruction to close the log up, so it won't be missed.
- Let's change the format of commit message example to code block highlight instead of inline.

 ````
9. Run `git log` to check if your changes have been automatically committed. To close the log, press `Q`.

    Here's what you should expect to see as your commit message:

    ```bash
    docs: add @your_username as a contributor
    ```
````

Reply:
Done